dumpling/export/sql_test.go

```diff
@@ -1747,6 +1747,29 @@ func TestPickupPossibleField(t *testing.T) {
 	}
 }
 
+func TestCheckIfSeqExists(t *testing.T) {
+	t.Parallel()
+
+	conn, err := db.Conn(context.Background())
```

Comment:
`db`, `mock` are not initialized:
```suggestion
	db, mock, err := sqlmock.New()
	require.NoError(t, err)
	defer func() {
		require.NoError(t, db.Close())
	}()

	conn, err := db.Conn(context.Background())
```

Reply:
sorry~ mistake while solving conflicts